decentraland/unity-explorer
# Feat: one scene loading

## What does this PR change?
Close #1217

- Add possibility to load only 1 scene (where Player is presented). It is controlled in `RealmPartitionSettings`
- Add possibility to load only 1 scene from the start (for worlds)
- Extended commands with additional parameter to load worlds
`/goto-local 5,7 only`
`/goto goerli 82,1 only`

## How to test the changes?

1. Launch the explorer
2. type `/goto goerli 82,1 only` and verify you get in Goerli with just 82,1 scene loaded.
3. type `/goto-local 77,1 only` and verify you are still in Goerli with just 77,1 scene loaded.
4. Jump to Genesis back (via map or command) and verify it load everything

tip: try to play with scenes loading radius in settings and verify it is not affecting when scene are loaded with `only` parameter from the chat

NOTE: see [previous check](https://github.com/decentraland/unity-explorer/pull/1253) of this PR
 
## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
